src/lib/es2020.bigint.d.ts

```diff
@@ -126,543 +126,23 @@ declare var BigInt: BigIntConstructor;
-     * argument in the next call to the callback function.
-     * @param callbackfn A function that accepts up to four arguments. The reduceRight method calls
-     * the callbackfn function one time for each element in the array.
-     * @param initialValue If initialValue is specified, it is used as the initial value to start
-     * the accumulation. The first call to the callbackfn function provides this value as an argument
-     * instead of an array value.
-     */
-    reduceRight<U>(callbackfn: (previousValue: U, currentValue: bigint, currentIndex: number, array: BigUint64Array) => U, initialValue: U): U;
-
-    /** Reverses the elements in the array. */
-    reverse(): this;
-
-    /**
-     * Sets a value or an array of values.
-     * @param array A typed or untyped array of values to set.
-     * @param offset The index in the current array at which the values are to be written.
-     */
-    set(array: ArrayLike<bigint>, offset?: number): void;
-
-    /**
-     * Returns a section of an array.
-     * @param start The beginning of the specified portion of the array.
-     * @param end The end of the specified portion of the array.
-     */
-    slice(start?: number, end?: number): BigUint64Array;
-
-    /**
-     * Determines whether the specified callback function returns true for any element of an array.
-     * @param predicate A function that accepts up to three arguments. The some method calls the
-     * predicate function for each element in the array until the predicate returns true, or until
-     * the end of the array.
-     * @param thisArg An object to which the this keyword can refer in the predicate function.
-     * If thisArg is omitted, undefined is used as the this value.
-     */
-    some(predicate: (value: bigint, index: number, array: BigUint64Array) => boolean, thisArg?: any): boolean;
-
-    /**
-     * Sorts the array.
-     * @param compareFn The function used to determine the order of the elements. If omitted, the elements are sorted in ascending order.
-     */
-    sort(compareFn?: (a: bigint, b: bigint) => number | bigint): this;
-
-    /**
-     * Gets a new BigUint64Array view of the ArrayBuffer store for this array, referencing the elements
-     * at begin, inclusive, up to end, exclusive.
-     * @param begin The index of the beginning of the array.
-     * @param end The index of the end of the array.
-     */
-    subarray(begin?: number, end?: number): BigUint64Array;
-
-    /** Converts the array to a string by using the current locale. */
-    toLocaleString(): string;
-
-    /** Returns a string representation of the array. */
-    toString(): string;
-
-    /** Returns the primitive value of the specified object. */
-    valueOf(): BigUint64Array;
-
-    /** Yields each value in the array. */
-    values(): IterableIterator<bigint>;
-
-    [Symbol.iterator](): IterableIterator<bigint>;
-
+interface BigUint64Array extends TypedArray<bigint> {

```

Comment:
I think there are many properties missing from `BigUint64Array` and `BigInt64Array` if we declare them that way.
From skimming the properties, I think what we want is for `BigInt64Array` to extend from `TypedArray<bigint>` and from `ArrayBufferView`?